event-stream-processing/src/inversify.config.ts

```diff
@@ -88,6 +89,7 @@ function create(): Container {
   myContainer.bind<Parser>(TYPES.PreParser).to(Wso2AccessParser);
   myContainer.bind<Parser>(TYPES.PreParser).to(TomcatParser);
   myContainer.bind<Parser>(TYPES.PreParser).to(RenameParser);
+  myContainer.bind<Parser>(TYPES.PreParser).to(IISParser);
```

Comment:
It probably is good to bump this before the rename parser. It's not "official" but it does return them in the order defined.

Reply:
bump up it before the rename parser